sips/sip-010/sip-010-fungible-token-standard.md

```diff
@@ -0,0 +1,132 @@
+## Preamble
+
+- Sip Number: 010
```

Comment:
Formatting -- `SIP Number`.  Also, we don't use bulleted lists for the Preamble section -- these should just be individual paragraphs.